pkgs/applications/networking/avalanchego/default.nix

```diff
@@ -40,6 +40,6 @@ buildGoModule rec {
     homepage = "https://github.com/ava-labs/avalanchego";
     changelog = "https://github.com/ava-labs/avalanchego/releases/tag/v${version}";
     license = licenses.bsd3;
-    maintainers = with maintainers; [ urandom ];
+    maintainers = with maintainers; [ urandom qjoly ];
```

Comment:
This change is unrelated to the version bump, so you should split it into a separate commit.

Reply:
I've always updated the package and added my name in a single commit.

I added a second line to the commit description.

Reply:
I've always suggested separating updates from the rest.

Reply:
I agree with @natsukium - the update should be a single commit and the added maintainer should be a single commit. Both can be in one PR of course!

Reply:
I take a coffee and I update the PR

Reply:
Awesome, thanks! That reminds me that I want another cup of coffee as well! :laughing:

Reply:
Thanks!